Specify pip3

jsf9k opened the pull request:
## 🗣 Description ##

This pull request modifies the `Dockerfile` to specify the use of `pip3`.

## 💭 Motivation and context ##

In this case it doesn't matter because we are starting from a Python3-specific base container, but other projects that use this skeleton may not be.  Specifying `pip3` ensures that the Python2 version of `pip` is not called by mistake.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

mcdonnnj reviewed (approved):
Seems reasonable enough.

dav3r reviewed (commented):
Good thinking. 👍

jmorrowomni committed:
Specify pip3

In this case it doesn't matter because we are starting from a
Python3-specific base container, but other projects that use this
skeleton may not be.  Specifying pip3 ensures that the Python 2
version of pip is not called by mistake.
